codespell_lib/data/dictionary.txt

```diff
@@ -13223,6 +13224,8 @@ conervative->conservative
 conesencus->consensus
 conet->connect
 coneted->connected
+conetent->content
+conetents->contents
```

Comment:
Perhaps add
```
conetented->contented
conetenting->contenting
```

Reply:
Thanks for pointing that out! In this context, "content" meant material or substance. I didn't even think of the other meaning of the word.